components/automate-cli/pkg/verifyserver/server/api/v1/handler.go

```diff
@@ -1,15 +1,17 @@
 package v1
 
 import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/batchcheckservice"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/statusservice"
 	"github.com/chef/automate/lib/logger"
 )
 
 type Handler struct {
-	Logger        logger.Logger
-	StatusService statusservice.IStatusService
-	BatchCheckService batchcheckservice.IBatchCheckService
+	Logger             logger.Logger
+	StatusService      statusservice.IStatusService
+	BatchCheckService  batchcheckservice.IBatchCheckService
+	MockServerServices []*models.Server
```

Comment:
You missed adding the StartMockService to this and adding a chain constructor method for this.

Reply:
Added